latex/chain.tex

```diff
@@ -8,7 +8,7 @@ \section{Blockchain layer}
 \newcommand{\Seede}{\mathsf{Seed}_\eta}
 \newcommand{\SlotsPrior}{\ensuremath{\mathsf{SlotsPrior}}}
 \newcommand{\StartRewards}{\ensuremath{\mathsf{StartRewards}}}
-\newcommand{\ActiveSlotCoeff}{\mathsf{ActiveSlotCoeff}}
+\newcommand{\activeSlotCoeff}[1]{\fun{activeSlotCoeff}~ \var{#1}}
```

Comment:
There are/were several instances of this. What does the non-breaking space `~` followed by a space mean?

Reply:
I'm not sure to be honest.